data/mods/Aftershock/items/gunmods/laser_gunmods.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "id": "electrolaser_conversion",
```

Comment:
What would be the reason for a mod that decreases damage but increases power consumption?

Reply:
Steep efficiency loss from making the gun behave in a way different than what its originally designed for.  (Plus thats how it works in vanilla too. )

Reply:
no no, I meant why gameplay wise WHY you'd want to have a weapon that costs more to fire, but does less damage?

Reply:
Ah it turns the laser into a ranged stunner.  Which has some niche combat uses, the most important one being that it lets you escape things you wouldn't be able to kill using the unmodded gun.

E.X. The Av-22 will be useless against a Spartan, but with this mod you can more easily escape by stunning it

Reply:
Ah that makes sense; thank you!